src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -372,7 +372,7 @@
             },
 
             createToggles: function() {
-                var toggles = document.querySelectorAll('.sf-toggle');
+                var toggles = document.querySelectorAll('a.sf-toggle');
```

Comment:
why forcing it on ``<a>`` elements ? You could keep the class-only selector and use ``hasClass(toggle)`` in your while loop